spec/features/consumer/shopping/embedded_groups_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+feature "Using embedded shopfront functionality", js: true do
+
+  Capybara.server_port = 9999
```

Comment:
It's not the firsr time I see this. Aren't we sure it's not already defined somewhere in the spec_helper.rb?